browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchFinalResult.kt

```diff
@@ -0,0 +1,41 @@
+package com.braintreepayments.api
+
+import android.net.Uri
+import org.json.JSONObject
+
+/**
+ * The result of a browser switch obtained from [BrowserSwitchClient.completeRequest]
+ */
+sealed class BrowserSwitchFinalResult {
+
+    /**
+     * The browser switch was successfully completed.
+     */
+    class Success internal constructor(
+        val deepLinkUrl: Uri,
```

Comment:
Nit: can we call this returnUrl since it could be app link or deep link now?

Reply:
Yeah def.

Reply:
Changed here: [ea9e3cf](https://github.com/braintree/browser-switch-android/pull/104/commits/ea9e3cfcbf3f60db3206c477cac50d22ff337063)